packages/components/organisms/f-header/package.json

```diff
@@ -40,13 +40,15 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "dependencies": {
-    "@justeat/f-services": "1.3.0"
+    "@justeat/f-services": "1.3.0",
+    "@justeat/f-wdio-utils": "0.1.0"
```

Comment:
I may have asked this before, but does this need to be in dependencies?

Reply:
nope definitely doesn't need to be in dependencies - I'll change that now and move to devDependencies 😄

Reply:
Thanks 😺